Automattic/crowdsignal-ui
# Include new Templates

## Summary

This PR includes a few more project templates:

- Customer Satisfaction Survey
- Event Ticket Registration
- Joblisting – Application Flow
- Join our Community
- Join Waitlist

Ref: pabtAt-1Vr-p2#comment-2106

Fixes Automattic/crowdsignal#433

## Testing Instructions

* Open the project page and check if all the listed templates were added and work as expected
